### DISCUSSION TITLE
Presenting certain trials in the timeline again

### DISCUSSION THREAD

**Discussion by prajwalgatti** [jspsych/jsPsych#854] [Q&A] [answered]
Hi all,

I was wondering if there is a way to go back several nodes in the experiment timeline and present them again? For a block of trials I want to fix a certain threshold for the number of trials in which user does not respond and if this condition is met, I would like for them to perform the block again.

My experiment contains several blocks and if this occurs in the 5th block, I would like to stop at the trial they are on, go to 1st trial of the 5th block and present again.

Please advise on how this can be done. Thank you!

**Comment by jodeleeuw** [comment, marked as answer]
Hi @prajwalgatti ,

This sounds like a canonical use case for [looping timelines](https://www.jspsych.org/overview/timeline/#looping-timelines)!

**Reply by prajwalgatti** [reply]
Ahh, yes! I'm sorry I missed that in the documentation. Thanks!
